src/borg/helpers/__init__.py

```diff
@@ -25,7 +25,14 @@
 from .parseformat import sizeof_fmt, sizeof_fmt_iec, sizeof_fmt_decimal
 from .parseformat import format_line, replace_placeholders, PlaceholderError
 from .parseformat import format_archive, parse_stringified_list, clean_lines
-from .parseformat import Location, location_validator, archivename_validator, comment_validator, text_validator
+from .parseformat import (
+    Location,
+    location_validator,
+    archivename_validator,
+    comment_validator,
+    text_validator,
+    relative_time_marker_validator,
+)
```

Comment:
I like most of the auto-formatting black is doing, but one thing I don't like: that it quickly tends to "vertically" format stuff, even if there is no real need.

In this case I guess we could easily avoid that, by either adding another `from .parseformat import ...` line or even just moving one imported name (e.g. Location) to an already existing import line (e.g. 27).